client/src/components/forms/form-wrapper.component.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { describe, expect, it } from 'vitest';
+import { FormWrapper } from './form-wrapper.component';
+import { render, screen } from '~/test/utils';
+
+describe('form-wrapper', () => {
+  it('should render form-wrapper on the page with child', () => {
+    render(<FormWrapper>Test Child</FormWrapper>);
+    expect(screen.getByText('Test Child')).toBeInTheDocument();
+  });
+});
```

Comment:
this is great, it definitely can take a bit of text, but that isn't really what this component is intended to do. I think it would be better to have a test that emulates a form, after all, that is the true purpose of this wrapper. It would also be useful in showing others how this component should be used.